Makefile

```diff
@@ -31,7 +31,9 @@ export MPI DEBUG
 # if you want to override this then just run make with
 # make COMPILER_TYPE=<your type>
 # where <your type> is gnu or intel
-ifeq "$(shell which ifort >/dev/null 2>&1; echo $$?)" "0" 
+ifneq ($(NERSC_HOST),)
```

Comment:
So this line checks for if you're on NERSC, but surely we want this to be applicable to whenever cray compilers are available? Does your fix still work if you replace the ifneq line with
```make
ifeq "$(shell which ftn >/dev/null 2>&1; echo $$?)" "0"
```

Reply:
Done, it still works on NERSC